docs/integration/cucumberjvm.adoc

```diff
@@ -2,12 +2,17 @@
 
 == Installation
 
-The available latest version of `allure-cucumber-jvm`:
+Each major version of Cucumber JVM requires particular version of Allure Cucumber JVM adapter.
```

Comment:
Sorry, very small change:
`Each major version of Cucumber JVM requires **a** particular version of Allure Cucumber JVM adapter.`

Reply:
Thanks!